src/Fable.Import.Electron.fs

```diff
@@ -22,24 +22,33 @@ module ElectronPrivate =
 
 module Electron =
     type [<Import("EventEmitter", "electron")>] EventEmitter() =
-        interface NodeJS.EventEmitter with
-            member __.addListener(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.on(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.once(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.removeListener(``event``: string, listener: Function): NodeJS.EventEmitter = failwith "JS only"
-            member __.removeAllListeners(?``event``: string): NodeJS.EventEmitter = failwith "JS only"
-            member __.setMaxListeners(n: int): unit = failwith "JS only"
+        interface Node.Events.event_types.EventEmitter with
+            member __.addListener(``event``: U2<string, Symbol>, listener: Function): Node.Events.event_types.EventEmitter = failwith "JS only"
```

Comment:
@alfonsogarciacaro When I had it only string the compiler made me specify.  Now I am wondering if I am running a wrong version of something based on this and your other comment.  Hoping I didn't butcher it for no reason.